jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -504,6 +505,47 @@ public FilterHolder addFilter(String filterClass, String pathSpec, EnumSet<Dispa
     {
         return getServletHandler().addFilterWithMapping(filterClass, pathSpec, dispatches);
     }
+    
+    /**
+     * Utility Method to allow for manual execution of {@link javax.servlet.ServletContainerInitializer} when using Embedded Jetty.
+     * @param containerInitializer the ServletContainerInitializer to register.
+     * @see Initializer
+     */
+    public void addServletContainerInitializer(ServletContainerInitializer containerInitializer)
```

Comment:
The style of other Jetty APIs that receive objects that are wrapped by `*Holder` classes is to return the holder (e.g. the various `addServlet()` in this class).

I would do the same here, change the return type from `void` to `ServletContainerInitializerHolder`.